db/migrations/2022120207400001-add-transfer-events-to-chain-events-table.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-prototype-builtins */
+const { QueryTypes } = require("sequelize");
+
+const { ChainEvents } = require("../models/chain-events.model");
+
+const {
+  Web3Connection,
+} = require("@taikai/dappkit");
+require("dotenv").config();
+
+const name = "add-transfer-events-to-chain-events-table";
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
```

Comment:
PS: Make sure that the event-name does not yet exist before adding it to the list